ci_framework/roles/ci_setup/tasks/packages.yml

```diff
@@ -15,46 +15,50 @@
     name: "{{ cifmw_ci_setup_packages }}"
     state: latest
 
+- name: Gather version of openshift client
+
+  register: _oc_version
+  environment: >-
+    {{
+      {'PATH': cifmw_path}
+      if cifmw_path is defined else
+      {}
+    }}
+  ansible.builtin.command:
+    cmd: "oc version -o yaml"
+  changed_when: false
+  failed_when: false
+
 - name: Install openshift client
   tags:
     - bootstrap
     - packages
+  when: >-
+     _oc_version.rc != 0 or
+     (
+      _oc_version | int >= cifmw_ci_setup_openshift_minimum_version
```

Comment:
hmmm that will most probably fail, the YAML answer is, at least with `--client`:
```YAML
clientVersion:
  buildDate: "2023-06-01T00:58:07Z"
  compiler: gc
  gitCommit: 05d83eff7e17160e679898a2a5cd6019ec252c49
  gitTreeState: clean
  gitVersion: 4.13.0-202305312300.p0.g05d83ef.assembly.stream-05d83ef
  goVersion: go1.19.9
  major: ""
  minor: ""
  platform: linux/amd64
kustomizeVersion: v4.5.7
releaseClientVersion: 4.13.3
```